common/models/container.js

```diff
@@ -220,17 +235,15 @@ module.exports = function(Container) {
                 Promise.map(orders,
```

Comment:
Instead of: `Promise.map(orders,`
could it be improved to be: `return Promise.map(orders,`

Unless ofcourse there are no more promise chains after this and everything is taken care of via callbacks like `next()`

Reply:
I dared not touch anything here lest it should break.